docs/troubleshooting/clean-install.md

```diff
@@ -1,6 +1,6 @@
 # How to clean install Streamlit
 
-If you run into any issues while installing Streamlit or if you just want to do a clean install, don't worry, we've got you covered. In this guide, we'll show you how to clean install Streamlit for Windows, macOS, and Linux.
```

Comment:
This feels like another usage that's "only" not "obvious"

Reply:
I don't feel like it adds anything here. For example, if the sentence were: "If you run into any issues while installing Streamlit, you want to do a clean install, or you JUST like wasting time", then the just emphasizes that we think you don't need to do it. But generally we do think people should use environments, doing a clean install for each project

Reply:
Okay, I'm convinced =)